### PR TITLE
Recorder

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR handles issue #300

### CODE REVIEW

File: templates/apply.html [hunk 1]
```diff
@@ -251,7 +251,7 @@ <h3>Upload your deed</h3>
 
                           <ol>
                             <li>If you <strong>do</strong> have a paper copy handy, scan it or take a picture of the first page.</li>
-                            <li>If you <strong>don't</strong> have a copy, download it from the <a href="http://162.217.184.82/i2/default.aspx" target="_blank">Cook County Recorder of Deeds</a> for $4.00 or pick it up in person at <a href='https://www.google.com/maps/place/Cook+County+Recorder+of+Deeds/@41.883823,-87.63144,17z' target='_blank'>118 N Clark St.</a></li>
+                            <li>If you <strong>don't</strong> have a copy, download it from the <a href="http://ccrecorder.org/" target="_blank">Cook County Recorder of Deeds</a> for a small fee or pick it up in person at <a href='https://www.google.com/maps/place/Cook+County+Recorder+of+Deeds/@41.883823,-87.63144,17z' target='_blank'>118 N Clark St.</a></li>
```
Comment (hancush) [inline templates/apply.html]: Why the loss of specificity?

File: templates/faq.html [hunk 2]
```diff
@@ -190,41 +190,11 @@ <h3 id='faq-block'>What defines a block?</h3>
...
-    </p>
-
-    <p><strong>4. Download your deed as a PDF (this costs $4.00)</strong></p>
-    <p class="deed-explain">
-    Within the pop-up, click Print Document and enter your payment information. Once you are done, you will have the scanned document as a PDF on your computer.</p>
-
-    <p>
-      <img class='img-responsive img-thumbnail' src="/static/images/deed3.png"/>
-    </p>
+    {% include "partials/deed_instructions.html" %}
```
Comment (hancush) [inline templates/faq.html]: Nice use of the partial :-)

File: templates/apply.html (hunk 1 above)
Comment (reginafcompton) [reply templates/apply.html]: The service that the CC Recorder uses now charges $3.00. So, the number can change: we should not hold ourselves accountable to tiny discrepancies, if the City modifies the cost again.

Comment (hancush) [reply templates/apply.html]: That sounds alright to me!

**Pull request merged**